raw_data/preprocessed/africa/namibia_english.txt

```diff
@@ -60,19 +60,19 @@ Functions of Environmental Commissioner
+(20) [1] For the purpose of this section “exceptional circumstances” includes circumstances in which the delay necessary to issue a written order that meets the requirements of subsection [2] would result in danger to human life or the environment. [2] An environmental officer may issue a compliance order to a person whom the environmental officer has reason to believe [a] has contravened this Act; or [b] has contravened a condition of an environmental clearance certificate. [3] A compliance order must set out [a] the name of the person to whom the order applies; [b] the provision or condition which has been contravened; [c] details of the nature and extent of the contravention; [d] any steps that are required to be taken and the period within which those steps must be taken; and [e] any penalty that may be imposed in terms of this Act if those steps are not taken; [f] the procedure to be followed in lodging an objection to the compliance order with the Minister; and [g] any other prescribed matter. [4] In exceptional circumstances a compliance order may be given orally, but within a period of seven days after such order is given, a written order must be issued in accordance with subsection [3]. [5] A person who receives a compliance order must comply with that order within the time period stated in the order unless the Minister has agreed to suspend the operation of the compliance order under section [21][6] Where a person fails to take any measures specified in the compliance order without raising an objection an environmental officer may take the measures or cause them to be taken. [7] Any costs incurred by the environmental officer in connection with any action taken under subsection [6] may be recovered from the person referred to in that subsection as a debt owing to the State. [8] Any person who, without good reason, fails or refuses to comply with a compliance order commits an offence and is liable on conviction to a fine not exceeding N$500 000 or to imprisonment for a period not exceeding 25 years or to both such fine and such imprisonment.
```

Comment:
Please remove the space in `N$500 000`. Please do the same with respect to all dollar amounts in this file.